### DISCUSSION TITLE
【求助】前几天在奥运会上看到了一位很可爱的女孩子，有谁有联系方式吗？

### DISCUSSION THREAD

**Discussion by obuyniq** [share121/inter-knot#3014] [其他]
![](https://cdn.luogu.com.cn/upload/image_hosting/j0e30uzu.png)

就是这位，太可爱了♥♥♥

**Comment by cpu644** [comment]
你这个美颜滤镜有点重啊

**Comment by lllliufeng** [comment]
我去，蓝精灵（

**Comment by Ruanachan** [comment]
md，神经病

**Comment by Goodname-LOL** [comment]
眼睛瞎了

**Reply by CyccyYan** [reply]
还挺涩

**Reply by Goodname-LOL** [reply]
关键一联想辣个蓝精灵，呜呜呜还我可爱的苍角！！TOT

**Comment by FWJ321** [comment]
别搞这种东西，原版是什么你又不是不知道

**Comment by yzysrewrwqr** [comment]
你是真饿了<img src="https://img-static.mihoyo.com/communityweb/upload/6b88559cbba96a1fa965e5412736370d.png" alt="兔躯一震" width="75" >

**Comment by Xy9223** [comment]
我勒个苍角啊<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >

**Comment by QHY8IV** [comment]
ybq不要搞这些有的没的

**Comment by chihung3927** [comment]
我觉得应该是蓝精灵吧

**Comment by isle200inf** [comment]
![OIP](https://github.com/user-attachments/assets/e6e39e5c-192b-47e2-8e6b-48ac16862e88)
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by lcwssys** [comment]
我去

**Comment by opqphp** [comment]
![Screenshot_2024-08-20-20-05-28-77_b783bf344239542886fee7b48fa4b892](https://github.com/user-attachments/assets/fea45c7d-6b8c-4e8c-b373-554438e70fdd)
用这个联系  名为苍角  艾莲卡池一堆（之后就退了也不知道多不多了）好多人都满命了<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by NZTWR** [comment]
离谱

**Comment by Bovkey** [comment]
这个委托我接了

**Reply by HFOxin** [reply]
你们脚本也太过分了，我最近一个委托都没接到

**Comment by HFOxin** [comment]
苍角被黑的最惨的一次

**Comment by Junjun280** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/9249bd36125b953ddd445b19151427d6.png" alt="安比 疑惑" width="75" >我嘞个苍角啊
